newdle_frontend.js

```diff
@@ -1,41 +1,74 @@
+/* Internal values */
 
+// To be changed with emojis in time.
 const TICK_STRING = "tick for now"
 
 const CROSS_STRING = "cross for now"
 
-const API_INFO = {headline: "Boris Johnson says tens of thousands could be sent to Rwanda under relocation plan for asylum seekers - UK politics live - The Guardian", scrambled_headline: "Bsrio onnJsoh asys snte fo onduathss uolcd be nset ot awnaRd rneud erolniocta pnla rof mluyas sesekre - UK iosplcti ievl - eTh andiuarG"}
+let headlineArray = []
 
-const HEADLINE_ARRAY = API_INFO.headline.split(" ")
-const HEADLINE_ARRAY_LOWER = API_INFO.headline.toLowerCase().split(" ")
+let headlineArrayLower = [] 
 
 const RESULT_PTAG = document.getElementById("result_text")
 
+const GUESS_BOX = document.getElementById("guess_box")
+
+const GUESS_BUTTON = document.getElementById("button")
+
+/* Making boxes unavailable */
+
+GUESS_BOX.disabled = true;
```

Comment:
You could either have the javascript set the boxes to disabled, or you could have the disabled property set in the HTML page already